ext/nnstreamer/tensor_filter/tensor_filter_python3.cc

```diff
@@ -593,7 +593,12 @@ PYCore::run (const GstTensorMemory *input, GstTensorMemory *output)
         break;
       }
     }
-
+    if (res != 0) {
+      for (auto &item : outputArrayMap) {
+        Py_SAFEDECREF (item.second);
+      }
+      outputArrayMap.clear ();
```

Comment:
Is it really ok to remove ALL from outputArrayMap?
or should be remove items inserted in this function only? (what is there is an outputArrayMap entry owned by a caller, which will call eventHandler later?)

Reply:
You're right. I'll fix it to only delete the problem part of outputArrayMap.

Reply:
Thank you, I've updated.